website/pages/payments.html

```diff
@@ -68,16 +66,10 @@
     </tr>
     </thead>
       {{ for(var p in it.stats.pools[pool].payments) { }}
-<tr>
+      <tr>
           <td>
-          {{if (String(it.stats.pools[pool].name).startsWith("zcash")) { }}
-          <a href="https://explorer.zcha.in/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else if (String(it.stats.pools[pool].name).startsWith("zclassic")) { }}
-            <a href="https://classic.zcha.in/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else if (String(it.stats.pools[pool].name).startsWith("hush")) { }}
-            <a href="https://explorer.myhush.org/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else if (String(it.stats.pools[pool].name).startsWith("zen")) { }}
-            <a href="http://node1.zenchain.info:8886/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
+          {{if (it.poolsConfigs[pool].explorerOptions.txExplorer) { }}
```

Comment:
Can you check to make sure this won't throw an error if `explorerOptions` doesn't exist and is undefined?